src/WebJobs.Script/Description/Rpc/WorkerLanguageInvoker.cs

```diff
@@ -63,7 +62,7 @@ protected override async Task<object> InvokeCore(object[] parameters, FunctionIn
             ScriptInvocationContext invocationContext = new ScriptInvocationContext()
             {
                 FunctionMetadata = Metadata,
-                BindingData = context.Binder.BindingData.Where(kvp => !(kvp.Value is DefaultHttpRequest)).ToDictionary(a => a.Key, b => b.Value),
+                BindingData = context.Binder.BindingData,   // This has duplicates too (of type DefaultHttpRequest). Needs to be removed after verifying this can indeed be constructed by the workers from the rest of the data being passed.
```

Comment:
Please open a github issue to address this issue and add link to the issue here.